### DISCUSSION TITLE
Add option to start herd just in background

### DISCUSSION THREAD

**Discussion by hebbet** [beyondcode/herd-community#817] [Ideas]
With 1.7.0 you changed "Herd dashboard now opens automatically when the app starts"

it would be nice to have an option for the old behaviour.

i liked that it started automatically when i login on my windows pc. 

i personally don't see any benefit for the launch of dashboard at every windows login.

**Comment by Bosphoramus** [comment]
Yeah, i agree, i think there should be an option to start Laravel Herd in the background, every time i turn on my PC i have to close the Herd window

**Comment by Brugman** [comment]
1.8.0 just added this.
